### PR TITLE
support ingress read timeout for azure spring app

### PR DESCRIPTION
**Pull request by frankliu20**
---
We will release a new version in #4844
Version and setup.py will be updated in #4844

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
spring

File: src/spring/HISTORY.md [hunk 1]
```diff
@@ -2,4 +2,9 @@ Release History
...
 1.0.0
 ---
-* Initialize extension "Spring" to manage Azure Spring Apps resources.
\ No newline at end of file
+* Initialize extension "Spring" to manage Azure Spring Apps resources.
+
+1.1.0
+---
+* New argument `az spring create` has new argument "--ingress-read-timeout" to set ingress read timeout when create Azure Spring App.
+* New argument `az spring update` has new argument "--ingress-read-timeout" to update ingress read timeout for Azure Spring App.
```
Comment (zhoxing-ms) [inline src/spring/HISTORY.md:10]: Please write the history notes of the new version at the top of the old version

Comment (zhoxing-ms) [inline src/spring/HISTORY.md:10]:
```suggestion
* New command `az spring create` has new argument "--ingress-read-timeout" to set ingress read timeout when create Azure Spring App.
* New command `az spring update` has new argument "--ingress-read-timeout" to update ingress read timeout for Azure Spring App.
```

Comment (frankliu20) [reply src/spring/HISTORY.md]: Thanks, Updated/

File: src/spring/HISTORY.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 Release History
 ===============
+1.1.0
+---
+* New argument `az spring create` has new argument "--ingress-read-timeout" to set ingress read timeout when create Azure Spring App.
+* New argument `az spring update` has new argument "--ingress-read-timeout" to update ingress read timeout for Azure Spring App.
```
Comment (zhoxing-ms) [inline src/spring/HISTORY.md:6]:
```suggestion
* New command `az spring create` has new argument "--ingress-read-timeout" to set ingress read timeout when create Azure Spring App.
* New command `az spring update` has new argument "--ingress-read-timeout" to update ingress read timeout for Azure Spring App.
```

**Review by zhoxing-ms — APPROVED** [review]

**Review by ShichaoQiu — APPROVED** [review]

**Pull request merged**